projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -71,7 +72,68 @@ context('Assisted Service Module', () => {
       cy.log(
         '--> Stop customer emulation using the end session button in the ASM UI'
       );
-      cy.get('cx-customer-emulation button').click();
+      cy.get('cx-customer-emulation [data-cy=logout]').click();
+      cy.get('cx-customer-emulation').should('not.exist');
+      cy.get('cx-customer-selection').should('exist');
+
+      cy.log('--> sign out and close ASM UI');
+      asm.agentSignOut();
+
+      cy.get('button[title="Close ASM"]').click();
+      cy.get('cx-asm-main-ui').should('exist');
+      cy.get('cx-asm-main-ui').should('not.be.visible');
+    });
+
+    it('agent should be able to bind anonymous cart to customer', () => {
+      let assignedCartId: string;
+      checkout.visitHomePage();
+      cy.get('cx-asm-main-ui').should('not.exist');
+
+      cy.log('--> Add to cart as an anonymous user');
+      cart.addProductAsAnonymous();
```

Comment:
Perhaps we can create an anonymous cart with api calls.  Just an idea. To be explored/discussed.